zhaquirks/develco/power_plug.py

```diff
@@ -0,0 +1,106 @@
+"""Smart Plugs."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.general import (
+    Alarms,
+    Basic,
+    DeviceTemperature,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.homeautomation import ElectricalMeasurement
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.smartenergy import Metering
+
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.develco import DEVELCO
+
+DEV_TEMP_ID = DeviceTemperature.attributes_by_name["current_temperature"].id
+
+
+class DevelcoDeviceTemperature(CustomCluster, DeviceTemperature):
+    """Device Temperature. Modify divisor."""
+
+    def _update_attribute(self, attrid, value):
+        if attrid == DEV_TEMP_ID and value is not None:
+            value = value * 100
+        super()._update_attribute(attrid, value)
+
+
+class SPLZB131(CustomDevice):
+    """Custom device Develco smart plug device."""
+
+    signature = {
+        MODELS_INFO: [(DEVELCO, "SPLZB-131")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: 0xC0C9,
+                DEVICE_TYPE: 1,
+                INPUT_CLUSTERS: [
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
```

Comment:
I'm guessing the removed endpoint 1 in the replacement did not have a function at all?